<launch>

    <node pkg="robosar_task_generator" type="robosar_task_graph_node" name="robosar_task_graph_node" output="screen" launch-prefix="gdb -ex run --args">
        <param name="eta" value="1.0"/>
        <param name="sampling_period" value="1"/>
        <param name="filter_threshold" value="7"/>
        <param name="map_topic" value="/map"/>
    </node>

    <node pkg="robosar_task_generator" type="frontier_filter.py" name="frontier_filter" output="screen">
        <param name="info_radius" value="1.0"/> 
        <param name="info_gain_threshold" value="0.3"/> 
        <param name="costmap_clearing_threshold" value="70"/>
        <param name="cluster_bandwidth" value="0.5"/> 
    </node>

</launch>